sdk/hybridkubernetes/arm-hybridkubernetes/review/arm-hybridkubernetes.api.md

```diff
@@ -116,7 +174,9 @@ export interface ConnectedClusterOperations {
 
 // @public
 export interface ConnectedClusterPatch {
-    properties?: Record<string, unknown>;
+    azureHybridBenefit?: AzureHybridBenefit;
```

Comment:
is this expected? can you show me the typespec for it?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/30594/checks?check_run_id=31632007393

Reply:
They added a new x-ms-client-flatten for this model. Currently, there's only two model `ConnectedCluster` and `ConnectedClusterPatch` will be impacted. The options for us are to either take this breaking for `ConnectedClusterPatch` or we break `ConnectedCluster`. Giving this flatten thing will have impacts to other models for scenarios like this, and we have decided that flatten is not what we want. I prefer to break `ConnectedCluster` instead.
Please turn off the flatten flag in m4 and regen,  @kazrael2119  thanks.